.github/workflows/run-tests.yml

```diff
@@ -11,14 +11,13 @@ on:
     branches:
       - "*"
   schedule:
-    - cron: '* 22 * * *'
+    - cron: '1 4 * * *'
```

Comment:
you're running this once every hour at 04:01?

Reply:
No, only once a day ;-)